frontend/src/components/pagedItemsTable/serializeFilterParamsTS.ts

```diff
@@ -4,128 +4,147 @@
 
 import { FILTER_TYPE } from '../../constants'
 import {
-	FilterSet,
-	isMetadataFilterValue,
-	isRangeFilterValue,
-	isStringArrayFilterValue,
-	isStringFilterValue,
-	MetadataFilterValue,
-	SortBy,
+    FilterDescription,
+    FilterOptions,
+    FilterSet,
+    FilterValue,
+    isMetadataFilterValue,
+    isRangeFilterValue,
+    isStringArrayFilterValue,
+    isStringFilterValue,
+    MetadataFilterValue,
+    SortBy,
 } from '../../models/paged_items'
 import { QueryParams } from '../../utils/api'
 import dayjs from 'dayjs'
 
-export default function serializeFilterParamsWithDescriptions(filters: FilterSet) {
-  const params: NonNullable<QueryParams> = {}
-
-  function hasSpaces(string) {
+function hasSpaces(string: string) {
```

Comment:
There was a doubt about the type there ? ;)

Reply:
Typescript doubted it ;)